src/containers/UserMenu/UserMenu.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { createAccount, signIn } from '../../actions'
+import { sendNewAccount, sendUserLogin } from '../../ApiFetch'
+export class UserMenu extends Component {
+  constructor() {
+    super()
+    this.state = {
+      name:'',
+      password:'',
+      email:''
+    }
+  }
+
+  handleChange = (e) =>{
+    this.setState({[e.target.name]: e.target.value})
+  }
+
+  submitForm = async (e) => {
+    e.preventDefault()
+    // await sendNewAccount({...this.state})
+    // await console.log(user)
+    
+    try {
+      const user = await sendUserLogin(this.state.email, this.state.password)
+      const results = await user.data
+      this.props.signIn(results)
+    } catch (error) {
+      this.setState({error})
+    }
+    // this.props.createAccount({thisstae})
+  }
+
+
+  render() {
```

Comment:
Should we split this up into a couple of components?

Reply:
Hmm, good idea.